rails_application/app/views/available_vat_rates/index.html.erb

```diff
@@ -13,14 +13,20 @@
     <tr>
       <th class="text-left py-2">Code</th>
       <th class="text-right py-2">Rate</th>
+      <th class="text-right py-2"></th>
     </tr>
   </thead>
 
   <tbody>
   <% @available_vat_rates.each do |available_vat_rate| %>
     <tr class="border-t">
       <td class="py-2"><%= available_vat_rate.code %></td>
-      <td class="py-2 text-right"><%= available_vat_rate.rate %></td>
+      <td class="text-right py-2"><%= available_vat_rate.rate %></td>
+      <td class="text-right py-2">
+        <%= form_with url: available_vat_rates_path, method: :delete do |f| %>
+          <%= f.hidden_field :vat_rate_code, value: available_vat_rate.code %>
+          <%= f.submit 'Delete' %>
+        <% end %>
```

Comment:
I used `form_with` instead of `link_to` because I needed to send `vat_rate_code`, which is our identifier. The `vat_rate_code` can contain a dot, e.g., `10.0`. Using `link_to` the code was added as a parameter to the url, which caused issues with Turbo. It appears that Turbo does not handle URLs with dots in the path correctly (see hotwired/turbo#608).

To avoid this issue, I decided to send the parameter through the form instead.